presto-jdbc/src/main/java/com/facebook/presto/jdbc/AbstractInfo.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.jdbc;
+
+import com.facebook.presto.spi.type.TypeSignature;
+
+import java.sql.Types;
+
+abstract class AbstractInfo
```

Comment:
We avoid using inheritance to share constants or static methods. Instead, we should make them public in `ColumnInfo` so that they can be reused (the class is package-private).